lmfdb/elliptic_curves/web_ec.py

```diff
@@ -313,8 +314,17 @@ def make_curve(self):
         # ell-adic Galois images:
 
         # remove adelic image record (prime set to 0) from ell-adic data if present
-        data['galois_data'] = [r for r in list(db.ec_galrep.search({'lmfdb_label': lmfdb_label})) if r["prime"] > 0]
-
+        galois_data = list(db.ec_galrep.search({'lmfdb_label': lmfdb_label}))
+        data['galois_data'] = [r for r in galois_data if r["prime"] > 0]
+        adelic_data = [r for r in galois_data if r["prime"] == 0]
+        if adelic_data:
+            assert len(adelic_data) == 1
+            my_adelic_data = adelic_data[0]
+            my_adelic_data['adelic_gens'] = ",".join([str(latex(dispZmat_from_list(z,2))) for z in my_adelic_data['adelic_gens']])
+            data['adelic_data'] = my_adelic_data
```

Comment:
I would probably just set `data['adelic_gens'] = ",".joing(...)`

Reply:
done (now called `adelic_gens_latex` to make clear it's a string)